mkdocs/docs/metrics/mock.md

```diff
@@ -28,18 +28,18 @@ kind: Metric
 metadata:
   name: request-count
 spec:
-	type: Counter
+  type: Counter
   mock:
-	- name: default
-	  level: "20.0"
-	- name: canary
-	  level: "15.0"
+  - name: default
+    level: "20.0"
+  - name: canary
+    level: "15.0"
 ```
 
 ## Explanation
-1. Use the `mock` field within metrics spec to indicate that the value of this metric needs to be mocked.
-2. The name within the `name-level` pairs must match the name of some version in the `versionInfo` field of the experiment.
+1. When the `mock` field within metrics spec is present, Iter8 will mock the values for this metric.
+2. The `name`s within the mock field must match names of versions in the `versionInfo` field of the experiment.
```

Comment:
This raises the question: what happens if no name matches? What are the semantics. We should document this as well; perhaps at the end.